### ISSUE TITLE
Video Track blank in Chrome/Safari/Firefox after iOS user backgrounds/foregrounds App in 

### ISSUE THREAD

**Issue by marcato15** [twilio/twilio-video.js#1693]
I work on a video calling service that has React Native apps for iOS, Android and is also able to be used via the browser.  When I have a call between a iOS App user and a Browser (Chrome or Safari), after the iOS user backgrounds the app during a call the video cuts out in browser (intended behavior) and we show the appropriate UI indicating the video feed has gone away. However, when the iOS user returns to the app the browser changes the UI back to showing the users video (indicating it is receiving the proper messaging that the video video has returned) **but the feed is blank.** There are no console log messages and if we do a 3 person call with someone on an Android device or another iOS device they see the video stream return fine. Also, if the Android user backgrounds the app and then returns the video feed restarts perfectly.  Sometimes this works the first time a user backgrounds the iOS app but will always not work by the second time the user backgrounds the app.  Seems to be a specific issue on the browser side and only when interacting with iOS Native apps. We only use P2P rooms.

 - [ x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x ] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [ x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [ x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Expected behavior:**

Video Feeds to not be blank after remote participant  backgrounds/foregrounds the app

**Actual behavior:**
Video Feeds is blank after remote participant  backgrounds/foregrounds the app

**Software versions:**

 - [x ] Browser(s): Chrome, Safari
 - [x ] Operating System: Mac, Windows
 - [x ] twilio-video.js: 2.16, (Twilio iOS Library 4.6.3)
 - [ x] Third-party libraries (e.g., Angular, React, etc.): React

**Comment by marcato15** [comment]
To follow up, a few things.

I consoled out the track when it is published and here is the Remote Video Track from the perspective of the Browser user. You'll not the dimensions are set to 2x2 and the muted property is set to false after the app backgrounds/foregrounds.

Initial Video Track
````
{
   "publication":{
      "track":{
         "sid":"MT739e7aee25dbbaa29afe04c324b7e063",
         "priority":"low",
         "isSwitchedOff":false,
         "isEnabled":true,
         "processor":null,
         "dimensions":{
            "height":640,
            "width":480
         },
         "processedTrack":null,
         "mediaStreamTrack":{
			contentHint: ""
			enabled: true
			id: "2e75e23C99e5f36E1B15FEB698ED7a0F"
			kind: "video"
			label: "2e75e23C99e5f36E1B15FEB698ED7a0F"
			muted: false
			onended: null
			onmute: null
			onunmute: null
			readyState: "live"
         },
         "isStarted":true,
         "name":"camera",
         "kind":"video"
      },
      "publishPriority":"standard",
      "kind":"video",
      "isTrackEnabled":true,
      "isSubscribed":true,
      "trackSid":"MT739e7aee25dbbaa29afe04c324b7e063",
      "trackName":"camera"
   }
}
````

Video track Publication after disconnecting

````
{
   "publication":{
      "track":{
         "sid":"MTfd7265beaa479808582fad73bb145001",
         "priority":null,
         "isSwitchedOff":false,
         "isEnabled":true,
         "processor":null,
         "dimensions":{
            "height":2,
            "width":2
         },
         "processedTrack":null,
         "mediaStreamTrack":{
			contentHint: ""
			enabled: true
			id: "bd8b6cf9-269e-4694-8fa1-e4222c9b904d"
			kind: "video"
			label: "bd8b6cf9-269e-4694-8fa1-e4222c9b904d"
			muted: true
			onended: null
			onmute: null
			onunmute: null
			readyState: "live"
         },
         "isStarted":true,
         "name":"camera",
         "kind":"video"
      },
      "publishPriority":"standard",
      "kind":"video",
      "isTrackEnabled":true,
      "isSubscribed":true,
      "trackSid":"MTfd7265beaa479808582fad73bb145001",
      "trackName":"camera"
   }
}
````

**Comment by marcato15** [comment]
So, I demoed the video ios app connecting to our browser and discovered the reason this doesn't appear in the app is b/c the demo app doesn't do anything to unpublish the video when you background the app so it just freezes the camera. We had set up logic to unpublish the video when a user backgrounds so the other participants don't just see a frozen frame and then republished the video when the app user returns. For some reason when we republish the video, the browser is not getting the feed even though the other apps do. Whats also interesting is the first time we background the app and return the feed comes back fine. But after that it never works.
